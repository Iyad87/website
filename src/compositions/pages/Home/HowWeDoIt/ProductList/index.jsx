import React from 'react';

import {
  Container,
  Image,
  Product,
  ProductColumn,
  Title,
  Link,
} from './styled';

import placeholder from './placeholder.svg';

const ProductList = () => (
  <Container>
    <ProductColumn>
      <Image src={placeholder} alt="our approach" />
      <Title>Our Approach</Title>
      <Product>Digital Transformation</Product>
      <Product>Engineering Consultancy</Product>
      <Product>Product Consultancy</Product>
      <Link to="/services#approach">More on Approach →</Link>
    </ProductColumn>
    <ProductColumn>
      <Image src={placeholder} alt="design & product" />
      <Title>Design &amp; Product</Title>
      <Product>Design Systems &amp; Design Operations</Product>
      <Product>Design Sprints</Product>
      <Product>User Interface Design &amp; Branding</Product>
      <Product>Research, Usability and User Experience</Product>
      <Link to="/services#design">More on Design →</Link>
    </ProductColumn>
    <ProductColumn>
      <Image src={placeholder} alt="engineering" />
      <Title>Engineering</Title>
      <Product>Engineering Leadership</Product>
      <Product>Back End &amp; Node.js</Product>
      <Product>Front End &amp; React.js</Product>
      <Product>DevOps &amp; Kubernetes</Product>
      <Product>Mobile &amp; React Native</Product>
      <Link to="/services#engineering">More on Engineering →</Link>
    </ProductColumn>
    <ProductColumn>
      <Image src={placeholder} alt="training" />
      <Title>Training</Title>
      <Product>Node Training</Product>
      <Product>React Training</Product>
      <Product>Design Systems Training</Product>
      <Product>UX Research Training</Product>
      <Product>UX Design Training</Product>
      <Link to="/services#training">More on Training →</Link>
    </ProductColumn>
  </Container>
);

export default ProductList;
