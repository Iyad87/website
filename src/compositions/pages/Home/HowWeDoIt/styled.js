import styled from 'styled-components';
import GatsbyLink from 'gatsby-link';
import { highlight } from '../../../../styles/colours';

export const Description = styled.span`
  display: block;
  margin-bottom: 1em;
`;

export const Link = styled(GatsbyLink)`
  color: ${highlight};
  text-decoration: none;

  &:hover {
    text-decoration: underline;
  }
`;
