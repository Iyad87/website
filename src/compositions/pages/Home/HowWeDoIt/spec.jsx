import React from 'react';
import { shallow } from 'enzyme';
import HowWeDoIt from './index';

test('compositions/homepage/HowWeDoIt snapshot', () => {
  const wrapper = shallow(<HowWeDoIt />);

  expect(wrapper).toMatchSnapshot();
});
