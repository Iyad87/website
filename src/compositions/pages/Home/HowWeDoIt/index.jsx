import React, { Fragment } from 'react';

import PageSection from '../../../../components/PageSection';
import PageSectionHeader from '../../../../components/PageSectionHeader';
import PageRule from '../../../../components/PageRule';
import ProductList from '../../../../components/ProductList';

import { Link, Description, ProductListWrapper } from './styled';

const HeaderDescription = (
  <Fragment>
    <Description>
      We thrive by working, thinking, and growing like a technology driven
      organisation. This is our mission, and our promise. Here’s how we do it:
    </Description>
    <Link to="/case-studies">Learn how we transform organisations →</Link>
  </Fragment>
);

const HowWeDoIt = () => (
  <PageSection>
    <PageSectionHeader description={HeaderDescription} title="How we do it" />
    <ProductListWrapper>
      <ProductList />
    </ProductListWrapper>
    <PageRule />
  </PageSection>
);

export default HowWeDoIt;
