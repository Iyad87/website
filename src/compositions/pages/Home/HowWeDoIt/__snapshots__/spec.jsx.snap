// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/homepage/HowWeDoIt snapshot 1`] = `
<styled.section>
  <PageSectionHeader
    description={
      <UNDEFINED>
        <styled.span>
          We thrive by working, thinking, and growing like a technology driven organisation. This is our mission, and our promise. Here’s how we do it:
        </styled.span>
        <Styled(GatsbyLink)
          to="/case-studies"
        >
          Learn how we transform organisations →
        </Styled(GatsbyLink)>
      </UNDEFINED>
    }
    title="How we do it"
  />
  <styled.div>
    <ProductList />
  </styled.div>
  <Styled(styled.hr) />
</styled.section>
`;
